crates/handlers/src/admin/v1/users/deactivate.rs

```diff
@@ -137,15 +138,37 @@ mod tests {
             serde_json::json!(state.clock.now())
         );
 
-        // It should have scheduled a deactivation job for the user
-        // XXX: we don't have a good way to look for the deactivation job
-        let job: Json<serde_json::Value> = sqlx::query_scalar(
-            "SELECT payload FROM queue_jobs WHERE queue_name = 'deactivate-user'",
-        )
-        .fetch_one(&pool)
-        .await
-        .expect("Deactivation job to be scheduled");
-        assert_eq!(job["user_id"], serde_json::json!(user.id));
+        // Make sure to run the jobs in the queue
+        state.run_jobs_in_queue().await;
```

Comment:
Shouldn't the jobs have finished running by this point?  If the job finishes after the response to `/deactivate` is generated, then it's possible for some side effects of the job to be missing from the response (namely the value of the `deactivated_at` timestamp).